docs/src/use/configure/configuration-files.md

```diff
@@ -159,15 +159,15 @@ export default [
 This configuration object applies to all JavaScript files except those ending with `.config.js`. Effectively, this is like having `files` set to `**/*`. In general, it's a good idea to always include `files` if you are specifying `ignores`.
 
 Note that when `files` is not specified, negated `ignores` patterns do not cause any matching files to be linted automatically.
-ESLint only lints files that are matched either by default or by a `files` pattern not ending with `/*` or `/**`.
+ESLint only lints files that are matched either by default or by a `files` pattern that is not `*` and does not end with `/*` or `/**`.
 
 ::: tip
 Use the [config inspector](https://github.com/eslint/config-inspector) (`--inspect-config` in the CLI) to test which config objects apply to a specific file.
 :::
 
 #### Specifying files with arbitrary extensions
 
-To lint files with extensions other than the default `.js`, `.cjs` and `.mjs`, include them in `files` with a pattern in the format of `"**/*.extension"`. (Any pattern that doesn't end with `/*` or `/**` will work.)
+To lint files with extensions other than the default `.js`, `.cjs` and `.mjs`, include them in `files` with a pattern in the format of `"**/*.extension"`. (Any pattern that is not `*` and doesn't end with `/*` or `/**` will work.)
```

Comment:
can we rather re phrase it to something like `any pattern that is * and ends with /* or /** wont work` rather? This sentence quite confuses me while reading everytime 😅.

Reply:
You are right, nobody talks like that. I removed the parentheses and reworded the sentence so it no longer contains a negated clause. Please, check again.